iot_verifier/src/settings.rs

```diff
@@ -2,6 +2,7 @@ use chrono::Duration;
 use config::{Config, Environment, File};
 use serde::Deserialize;
 use std::path::Path;
+use std::time::Duration as StdDuration;
```

Comment:
when we have multiple types of duration in the same module should we be consistent with which ones we alias and which ones we take from the original name? in the other module we're aliasing `chrono::Duration as ChronoDuration` but here we're flipping that

Reply:
I took the approach of aliasing the minority usage